docs/user-guide/release-notes/2.6.x.rst

```diff
@@ -25,6 +37,11 @@ New Features
 
 .. _RabbitMQ: https://www.rabbitmq.com/
 
+Bug Fixes
+---------
+
+This release has fixes for `multiple bugs <https://pulp.plan.io/issues?utf8=%E2%9C%93&set_filter=1&f%5B%5D=cf_4&op%5Bcf_4%5D=%3D&v%5Bcf_4%5D%5B%5D=2.6.0&f%5B%5D=tracker_id&op%5Btracker_id%5D=%3D&v%5Btracker_id%5D%5B%5D=1&f%5B%5D=&c%5B%5D=project&c%5B%5D=tracker&c%5B%5D=status&c%5B%5D=priority&c%5B%5D=cf_5&c%5B%5D=subject&c%5B%5D=author&c%5B%5D=assigned_to&c%5B%5D=cf_3&group_by=>`_.
```

Comment:
This needs to be moved to its own PR and that needs to be merged into 2.6-testing. Otherwise the 2.6.0 release notes will still point to the wrong bug tracker. The content looks right, just not in this PR.